pkg/fleetautoscalers/controller.go

```diff
@@ -152,6 +150,22 @@ func NewController(
 	return c
 }
 
+// NewExtensions binds the handlers to the webhook outside the initialization of the controller
+// initializes a new logger for extensions.
+func NewExtensions(wh *webhooks.WebHook) *Extensions {
+	ext := &Extensions{}
+
+	ext.baseLogger = runtime.NewLoggerWithSource("FleetAutscalerExtensions")
```

Comment:
This should use NewLoggerWithType

Reply:
I changed this one as well as the other `NewExtensions`.